packages/block-library/src/embed/util.js

```diff
@@ -174,8 +173,8 @@ export function getClassNames(
 			const potentialRatio = ASPECT_RATIOS[ ratioIndex ];
 			if ( aspectRatio >= potentialRatio.ratio ) {
 				return classnames( existingClassNames, {
-					[ potentialRatio.className ]: allowResponsive,
-					'wp-has-aspect-ratio': allowResponsive,
+					[ potentialRatio.className ]: true,
+					'wp-has-aspect-ratio': true,
```

Comment:
What should we know about this change?

Reply:
At the start of this function there is a check:
```
if ( ! allowResponsive ) { .... return ...; }
```

So this is the path for being responsive and doesn't need to take the value from variable. And I believe this is clearer to what this code does.

Reply:
You're totally right, I see it now.

Btw, a tip: `classNames` accepts as many strings as needed, so in this case where we always have `true` we could rewrite as:

```js
classNames( existingClassNames, potentialRatio.className, 'wp-has-aspect-ratio' )
```

Reply:
Good tip! Thanks! Changed it.